src/dbwrapper.h

```diff
@@ -284,6 +287,9 @@ class CDBWrapper
 
     bool WriteBatch(CDBBatch& batch, bool fSync = false);
 
+    // Get an estimate of LevelDB memory usage (in bytes).
+    size_t DynamicMemoryUsage();
```

Comment:
Could be `const`?

Reply:
Maybe it's not possible, since [`GetProperty` is not const](https://github.com/google/leveldb/blob/41172a24016bc29fc795ed504737392587f54e3d/include/leveldb/db.h#L121)

Reply:
Right, unless you want to give into the temptation of const cast.

Reply:
Nah, IMO it's fine as it is.

Reply:
> Maybe it's not possible, since GetProperty is not const

FWIW, it is possible to declare this const, because GetProperty is called on a pointer to a non-const object. Adding const here only makes the pointer address const, not the object located at that address.